tests/e2e/test_rosacli_cluster.go

```diff
@@ -297,4 +297,216 @@ var _ = Describe("Edit cluster",
 				Expect(err).To(BeNil())
 				Expect(stdout.String()).To(ContainSubstring(fmt.Sprintf("Creating cluster '%s' should succeed", clusterName)))
 			})
+		It("Check the basic validation for the classic rosa cluster creation by the rosa cli - [id:38770]", labels.High, labels.NonHCPCluster, func() {
+			invalidClusterNames := []string{
+				"1-test-1",
+				"-test-cluster",
+				"test-cluster-",
+			}
+			invalidMachineType := "not-exist-machine-type"
+			invalidReplicasErrorMapSingeAZ := map[string]string{
+				"1":  "requires at least 2 compute nodes",
+				"0":  "requires at least 2 compute nodes",
+				"-1": "must be non-negative",
+			}
+			invalidRegion := "not-exist-region"
+
+			invalidReplicasErrorMapMultiAZ := map[string]string{
+				"2":  "Multi AZ cluster requires at least 3 compute nodes",
+				"0":  "Multi AZ cluster requires at least 3 compute nodes",
+				"-3": "must be non-negative",
+				"5":  "Multi AZ clusters require that the number of compute nodes be a multiple of 3",
+			}
+			invalidReplicasTypeErrorMap := map[string]string{
+				"4.5":  "invalid argument \"4.5\" for \"--replicas\" flag",
+				"five": "invalid argument \"five\" for \"--replicas\" flag",
+			}
+			rosalCommand, err := config.RetrieveClusterCreationCommand(ciConfig.Test.CreateCommandFile)
```

Comment:
If this is a day1-negative case, we need a discussion about the day1 negative cases. Combine it to current CI job or have another job. If we pick the second resolution, then there won't be creation command file existing.